odoo/addons/base/views/res_users_views.xml

```diff
@@ -52,27 +52,6 @@
             <field name="binding_model_id" ref="base.model_res_users"/>
         </record>
 
-        <record id="identity_check_wizard" model="ir.ui.view">
```

Comment:
Don't forget the migration script to remove the view :) (hence red runbot by the way)